### ISSUE TITLE
Customizing shortcuts does not work.

### ISSUE THREAD

**Issue by kareltucek** [lydell/LinkHints#22]
Hi and thanks for developing this extension! 

Unfortunately, I seem to have run into issues while trying to customize the shortcuts:

1. Open the `moz-extension://.../options.html` page.
2. Locate `Main keyboard shortcuts` section.
3. Click "+" on the right.
  (`Press the keyboard shortcut you’d like to use!` hint shows up)
4. Tap a shortcut. E.g., `Ctrl+a` or `Alt+a`.
  (Nothing happens.)
5. Try some more shortcuts.
  (Shortcuts perform they browser role, e.g., `Ctrl+W` closes the tab, `Ctrl+R` refreshes it.)
  (So lets try some workaround.)
7. Notice `Export` button on right.
8. Export the config and try to customize it by hand.
  (Contains only `{}`, which unfortunately does not help.)

------

The main functionality (e.g., pressing `Alt+L` and hint) works fine.

### Debug info

```json
{
  "version": "1.1.0",
  "browser": "firefox",
  "userAgent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:79.0) Gecko/20100101 Firefox/79.0",
  "browserInfo": {
    "name": "Firefox",
    "vendor": "Mozilla",
    "version": "79.0",
    "buildID": "20200720193547"
  },
  "platformInfo": {
    "os": "linux",
    "arch": "x86-64"
  },
  "storage.sync": {
    "normal.alt-j": null
  },
  "storage.local": {
    "tutorialShown": true
  },
  "language": "en-US",
  "layout": null
}
```
Namely, it is up-to-date ubuntu+x11+i3-wm window manager.

**Comment by lydell** [comment]
I haven’t used the settings UI in a while so I haven’t noticed it breaking! It seems like if you refresh the page it suddenly starts working. Thanks for reporting!

**Comment by kareltucek** [comment]
> It seems like if you refresh the page it suddenly starts working.

Confirming. Thanks!

**Comment by lydell** [comment]
Fixed in version 1.1.1, which will be available as soon as Google and Mozilla have reviewed and approved the version.
